src/configure/activate.ts

```diff
@@ -27,15 +22,12 @@ export async function activateConfigurePipeline(context: vscode.ExtensionContext
     // The command has been defined in the package.json file
     // Now provide the implementation of the command with registerCommand
     // The commandId parameter must match the command field in package.json
-    vscode.commands.registerCommand('configure-pipeline', async (node: any) => {
+    registerCommand('configure-pipeline', async (actionContext: IActionContext, node: any) => {
         // The code you place here will be executed every time your command is executed
-        await configurePipeline(node);
+        let telemetryHelper = new TelemetryHelper(actionContext, 'configure', journeyId);
```

Comment:
Why does telemetryHelper take 'configure' as an input? What if there are multiple commands - will there be multiple helper instances?

Reply:
Each command will be registered seperately, and at a time only one command can be invoked. Thus everytime a new command is invoked, we will get a new actionContext with which we create a new telemetry helper.

Reply:
This looks weird to me. If we add another command to this extension, we will have one more telemetryHelper object. Would it make sense to define actionContext as a property in TelemetryHelper instead of in constructor and set this property whenever a new command is activated?